packages/testing/src/bdd/when.ts

```diff
@@ -62,6 +62,7 @@ export const runInWorker = (testOrContext: Context, workerDeclaration: (...insta
       const runner = new WorkerRunner(
         false,
```

Comment:
I already asked myself if we should "merge" the 2/3 boolean values (skipProvisioning, skipBootstrap and now grabAllTraffic) into a object (RunOptions or RunContext) ?

Reply:
👍 RunOptions